CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 #
 
 cmake_minimum_required(VERSION 3.0.0)
-project(WABT VERSION 1.0.13)
+project(WABT VERSION 1.0.21)
```

Comment:
Why jump from 13 to 21? Why not 14?

Reply:
I think this file was just very out of date.  1.0.20 is the last release.